internal/controller/rbac/namespace/reconciler.go

```diff
@@ -210,9 +213,15 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		}
 
 		log.Debug("Applied RBAC Role")
+		applied = append(applied, rl.GetName())
 	}
 
-	r.record.Event(ns, event.Normal(reasonApplyRoles, "Applied RBAC Roles"))
+	sort.Strings(applied)
+	if len(applied) > 3 {
```

Comment:
is this > 3 pattern pretty common in upstream k8s? i feel like i may have seen this before - not 💯 on it though, as I have hazy memories of thinking to myself "well why didn't you just tell me the rest of them too then?" 😂 

more curious than anything - not blocking from me 😁

Reply:
I think we use something like that in OpenShift. No strong opinions. I thought 3 is a good number :) 10 might be too.